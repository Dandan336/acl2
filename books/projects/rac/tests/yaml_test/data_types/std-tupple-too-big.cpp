#include <tuple>

// grrr
using namespace std;

// RAC begin

tuple<int, int, int, int, int, int, int, int, int> foo(int a)
{
  return tuple<int, int, int, int, int, int, int, int, int>(a, a, a, a, a, a, a, a, a);
}

// RAC end
