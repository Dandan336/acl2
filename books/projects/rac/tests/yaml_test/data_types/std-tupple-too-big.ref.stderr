std-tupple-too-big.cpp:8: syntax error, unexpected ',', expecting '>'
